pennylane/ops/qubit/matrix_ops.py

```diff
@@ -62,7 +63,7 @@ class QubitUnitary(Operation):
     grad_method = None
     """Gradient computation method."""
 
-    def __init__(self, *params, wires, do_queue=True):
+    def __init__(self, *params, wires, do_queue=True, unitary_check=True):
```

Comment:
```suggestion
    def __init__(self, *params, wires, do_queue=True, unitary_check=False):
```

I think we discussed the idea for unitarity checks at the start of Q3 and decided it was too much overhead. I think this should be defaulted to off, and only enabled if a user wants verification of the workload.